### DISCUSSION TITLE
c'è un modo per farlo funzionare in background ? (is there a way to make it work in the background?)

### DISCUSSION THREAD

**Discussion by simonevetere** [innoveit/react-native-ble-manager#993] [Q&A]
sto utilizzando questo modulo per connettermi ad un device low energy, esiste un modo per mantenere l'applicazione viva in background, e continui a eseguire task ? su IOS e su Android?
===========================================================
I'm using this module to connect to a low energy device, is there a way to keep the application alive in the background, and continue executing tasks? on IOS and on Android?

**Comment by lucasmrau** [comment]
Any help about this?

**Comment by marcosinigaglia** [comment]
Quick answer, no. There is no way to run tasks in iOS, in Android there are foreground services. In iOS the app can stay in scan and receive scanned devices and/or connect to a device even if the app is in the background. After the event, however, you have a few seconds of execution and then the app goes back to sleep.

**Reply by lucasmrau** [reply]
What about the Core Bluetooth for IOS?

**Reply by vhakulinen** [reply]
On iOS, you can instruct the system to keep the connection alive and receive peripheral's notifications while in background. See https://developer.apple.com/library/archive/documentation/NetworkingInternetWeb/Conceptual/CoreBluetooth_concepts/CoreBluetoothBackgroundProcessingForIOSApps/PerformingTasksWhileYourAppIsInTheBackground.html
